templates/syfosoknader/arbeidstakere.hbs

```diff
@@ -21,12 +21,12 @@
                 {{/if}}
                 {{# if innsendtDato}}
                     <p>
-                        Sendt til NAV {{ iso_to_date innsendtDato }}
+                        Sendt til NAV<br/>{{ iso_to_date innsendtDato }}
```

Comment:
skal ikke dette være sendtTilNavDato?

Reply:
Fiksa

Reply:
Dette er litt skummelt. Frilanser/selvstendig-søknaden som allerede kjører i prod, bruker innsendtDato. Så hvis du skal endre variabelnavnet her er du nødt til å splitte opp i to forskjellige request-objekter i syfogsak.

Reply:
Kim har rett, det heter innsendtDato ikke sendtTilNavDato... Jeg endrer tilbake :P